neuraxle/steps/numpy.py

```diff
@@ -37,34 +37,59 @@ def transform_one(self, data_inputs):
         return data_inputs.flatten()
 
 
-class NumpyConcatenateInnerFeatures(NonFittableMixin, BaseStep):
-    def __init__(self):
+class NumpyConcatenateOnCustomAxis(NonFittableMixin, BaseStep):
+    """
+    Numpy concetenation step where the concatenation is performed along the specified custom axis.
+    :param axis: the axis where the concatenation is performed.
+    :return: NumpyConcatenateOnCustomAxis instance which is inherited by base step.
+    """
```

Comment:
This docstring should be like this: 
```
    """
    Numpy concatenation step where the concatenation is performed along the specified custom axis.
    """
```

I might be wrong, but I think that the following part should instead be in another docstring in the constructor to be uniform with the other classes' documentation : 
```
    """
    Create a concatenator where the concatenation is performed along the specified custom axis. 

    :param axis: the axis where the concatenation is performed.
    :return: NumpyConcatenateOnCustomAxis instance which is inherited by base step.
    """
```

Note: I'm not sure the IDE would create a `    :return:` param for documentation here, perhaps.

Reply:
Will change it, thanks!